alvr/system_info/src/lib.rs

```diff
@@ -37,7 +37,7 @@ pub enum Platform {
     Lynx,
     SamsungGalaxyXR,
     AndroidUnknown,
-    AppleHeadset,
```

Comment:
This really falls out of the naming scheme, except the Pc platforms most of these are about the general platform, not the os. Maybe `AppleVision`? Either way what's the thought behind the naming change?